### PR TITLE
refactor: replace all references of CustomError in core

### PR DESCRIPTION
**Pull request by saikumarrs**
## Description of the change

Replaced all the references of CustomError in the transformer core files.
Along with it, deleted deprecated modules.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

N/A

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: src/versionedRouter.js [hunk 1]
```diff
@@ -217,7 +217,7 @@ async function handleDest(ctx, version, destination) {
 
   const events = ctx.request.body;
   if (!Array.isArray(events) || events.length === 0) {
-    throw new CustomError("Event is missing or in inappropriate format", 400);
+    throw new PlatformError("Event is missing or in inappropriate format");
```
[the remark this answers is not in the document]
Comment (koladilip) [reply src/versionedRouter.js:220]: we can tackle this later

**Review by koladilip — APPROVED** [review]

File: src/v0/util/index.js [hunk 2]
```diff
@@ -210,35 +214,6 @@ const getHashFromArrayWithValueAsObject = (
   return hashMap;
 };
 
-// NEED to decouple value finding and `required` checking
-// NEED TO DEPRECATE
-const setValues = (payload, message, mappingJson) => {
```
Comment (krishna2020) [inline src/v0/util/index.js:215]: How do we make sure the methods & classes being removed are not referred anywhere in cases of missing test coverage ?

Comment (saikumarrs) [reply src/v0/util/index.js:215]: I'm making sure these are not being used by blanket search throughout the code.

Comment (saikumarrs) [reply src/v0/util/index.js:215]: For now all the tests will fail in these individual PRs but we'll address all of that at once in the end.

**Review by krishna2020 — APPROVED** [review]

**Pull request merged**